src/fetch-wrapper.ts

```diff
@@ -153,7 +153,14 @@ export default function fetchWrapper(
 async function getResponseData(response: Response) {
   const contentType = response.headers.get("content-type");
   if (/application\/json/.test(contentType!)) {
-    return response.json();
+    // In the event that we get an empty response body we fallback to
+    // using .text(), but this should be investigated since if this were
+    // to occur in the GitHub API it really should not return an empty body.
+    try {
+      return response.json();
```

Comment:
You would likely need to use an `await` here in other for the `try` to actually catch anything since the error is presumably in the async phase of `.json()` rather than the sync execution.

Reply:
yes! good catch!

Reply:
We should cover this with a test after all instead of ignoring the line for coverage, this is too easy to miss.